webauthn-rs/src/lib.rs

```diff
@@ -1225,3 +1225,16 @@ impl Webauthn {
         self.core.authenticate_credential(reg, &state.ast)
     }
 }
+
+#[test]
+/// Test building a webauthn object from a chrome extension origin
+fn test_webauthnbuilder_chrome_url() {
```

Comment:
does the test pass or fail? 🤔

Reply:
it passes! and now the comment mentions it.